apps/studio/Dockerfile

```diff
@@ -31,22 +33,25 @@ RUN PUPPETEER_SKIP_DOWNLOAD=true pnpm install
 
 # Build the project
 COPY --from=builder /app/out/full/ .
-RUN PUBLIC_URL=${BASE_URL_PLACEHOLDER} NEXT_CONFIG_OUTPUT=export pnpm run build:studio
+RUN PUBLIC_URL=${BASE_URL_PLACEHOLDER} NEXT_BASE_PATH=${BASE_PATH_PLACEHOLDER} NEXT_CONFIG_OUTPUT=export pnpm run build:studio
  
 
 FROM docker.io/library/nginx:1.25.5-alpine AS runtime
 
 ARG BASE_URL_PLACEHOLDER
+ARG BASE_PATH_PLACEHOLDER
 # The base Nginx image automatically executes all shell scripts 
 # within the /docker-entrypoint.d/ directory ("entrypoint scripts")
 # when the container is started. See the relevant logic at
 # https://github.com/nginxinc/docker-nginx/blob/master/entrypoint/docker-entrypoint.sh#L16.
 ARG ENTRYPOINT_SCRIPT=/docker-entrypoint.d/set-public-url.sh
 
-COPY --from=installer /app/apps/studio/build /usr/share/nginx/html/
+COPY --from=installer --chown=nginx:nginx /app/apps/studio/build /usr/share/nginx/html/
 # Add an entrypoint script that replaces all occurrences of the 
-# placeholder value by the configured base URL. If no base URL
+# placeholder values by the configured base URL. If no base URL
 # is configured we assume the application is running at '/'.
-RUN echo "find /usr/share/nginx/html/ -type f -print0 | xargs -0 sed -i \"s|${BASE_URL_PLACEHOLDER}|\${BASE_URL}|g\"" > $ENTRYPOINT_SCRIPT && chmod +x $ENTRYPOINT_SCRIPT
+RUN echo "find /usr/share/nginx/html/ -type f -print0 | xargs -0 sed -i \"s|${BASE_URL_PLACEHOLDER}|\${BASE_URL}|g\"" >> $ENTRYPOINT_SCRIPT && \
```

Comment:
Should the URL Placeholder be removed then?

Reply:
@Shurtu-gal Thanks for the review. This was a good call, I simplified this PR and just pushed 72a94705ec306e687c04bc40280000f2590cbb09? 

Could you please have another look? (tested with success with and without a `BASE_URL` env variable)